src/ApprenticeAan/SFA.DAS.ApprenticeAan.Application/CalendarEvents/Queries/GetCalendarEventById/CalendarEvent.cs

```diff
@@ -0,0 +1,26 @@
+﻿using SFA.DAS.ApprenticeAan.Application.Entities;
+
+namespace SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+
+public class CalendarEvent
+{
+    public Guid CalendarEventId { get; set; }
+    public string CalendarName { get; set; } = null!;
+    public string EventFormat { get; set; } = string.Empty;
+    public DateTime StartDate { get; set; }
+    public DateTime EndDate { get; set; }
+    public string Description { get; set; } = string.Empty;
+    public string? Summary { get; set; }
+    public int? RegionId { get; set; }
+    public string? Location { get; set; }
+    public string? Postcode { get; set; }
+    public double? Latitude { get; set; }
+    public double? Longitude { get; set; }
+    public string? EventLink { get; set; }
+    public string ContactName { get; set; } = string.Empty;
+    public string ContactEmail { get; set; } = string.Empty;
+    public bool IsActive { get; set; }
+    public string? CancelReason { get; set; } = null!;
+    public IEnumerable<Attendee> Attendees { get; set; } = null!;
+    public IEnumerable<EventGuest> EventGuests { get; set; } = null!;
+}
```

Comment:
I've added properties to match the `CalendarEvent` class in the inner API. I've also renamed the `Start`, `End`, and `IsAttending` properties to `StartDate`, `EndDate`, and `IsActive`, to match the names in the inner API.

When RestEase deserialises the response on line 22 of `GetCalendarEventByIdQueryHandler`, it doesn't assign to these properties since the names are different. Now that I've changed the names to match the inner API, this is no longer a problem.

Reply:
The name is wrong in the inner api